dm/dm/worker/source_worker.go

```diff
@@ -625,6 +620,19 @@ func (w *SourceWorker) OperateSubTask(name string, op pb.TaskOp) error {
 	}
 
 	w.l.Info("OperateSubTask start", zap.Stringer("op", op), zap.String("task", name))
+	failpoint.Inject("SkipRefreshFromETCDInUT", func(_ failpoint.Value) {
+		failpoint.Goto("bypassRefresh")
+	})
+	if op == pb.TaskOp_Resume || op == pb.TaskOp_AutoResume {
```

Comment:
when user want to restart or start a stopped task, use the newest cfg from etcd.

dynamic update will impl later